src/test/scala/scala/tools/refactoring/tests/analysis/CompilationUnitDependenciesTest.scala

```diff
@@ -792,7 +788,26 @@ class CompilationUnitDependenciesTest extends TestHelper with CompilationUnitDep
         @throws(classOf[IOException])
         def read() = in.read()
       }
-    """)
+    """
+    val deps1 = """java.io.BufferedReader
+         java.io.FileReader
+         java.io.IOException
+         scala.this.Predef.String
+         """
+    val deps2 = deps1 + "\n" + scala.this.Predef.classOf
```

Comment:
I want to structure the test so it works before and after the commit in the
compiler.
On Thu, 10 Sep 2015 at 7:40 pm Simon Schäfer <notifications@github.com>
wrote:

> In
> src/test/scala/scala/tools/refactoring/tests/analysis/CompilationUnitDependenciesTest.scala
> <https://github.com/scala-ide/scala-refactoring/pull/96#discussion_r39141571>
> :
>
> > @@ -792,7 +788,26 @@ class CompilationUnitDependenciesTest extends TestHelper with CompilationUnitDep
> >          @throws(classOf[IOException])
> >          def read() = in.read()
> >        }
> > -    """)
> > +    """
> > +    val deps1 = """java.io.BufferedReader
> > +         java.io.FileReader
> > +         java.io.IOException
> > +         scala.this.Predef.String
> > +         """
> > +    val deps2 = deps1 + "\n" + scala.this.Predef.classOf
>
> Why didn't you just add this dependency to the previous list of
> dependencies?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scala-ide/scala-refactoring/pull/96/files#r39141571>.
>